examples/dynamo/vgg16_fp8_ptq.py

```diff
@@ -216,16 +216,7 @@ def calibrate_loop(model):
 
 with torch.no_grad():
     with export_torch_mode():
-        # Compile the model with Torch-TensorRT Dynamo backend
-        input_tensor = images.cuda()
-        exp_program = torch.export.export(model, (input_tensor,))
-        trt_model = torchtrt.dynamo.compile(
-            exp_program,
-            inputs=[input_tensor],
-            enabled_precisions={torch.float8_e4m3fn},
-            min_block_size=1,
-            debug=False,
-        )
```

Comment:
Leave these changes as is (use torch.export workflow for the tutorial). I would say just change the title "Torch Compile VGG16 with FP8 and PTQ" -> "Deploying Torch-TRT programs in FP8 precision". In the comments, you can write the same workflow applies to torch.compile frontend as well.